source/dem/insertion_file.cc

```diff
@@ -36,12 +40,25 @@ InsertionFile<dim>::insert(
           ++this->current_inserting_particle_type);
     }
 
-
   if (remaining_particles_of_each_type > 0)
     {
+      if (this->mark_for_update)
+        {
+          this->find_in_clearing_box_cells(triangulation);
+          this->mark_for_update = false;
+        }
+      if (this->clearing_particles)
+        {
+          this->clear_particle_box(particle_handler);
+        }
+
       // Read the input file
       std::map<std::string, std::vector<double>> particles_data;
-      fill_vectors_from_file(particles_data, this->file_name, ";");
+      fill_vectors_from_file(particles_data,
+                             insertion_files.at(current_file_id),
+                             ";");
```

Comment:
Again, I would use as default ","

Reply:
Good for now. Until we have a clearly documented standard, let'S not duplicate stuff.